config/routes.rb

```diff
@@ -79,11 +82,10 @@
   delete '/openid_connect/authorize' => 'openid_connect/authorization#destroy',
          as: :openid_connect_deny
 
-  get '/openid_connect/certs' => 'openid_connect/certs#index'
-
-  post '/openid_connect/token' => 'openid_connect/token#create'
-
-  get '/openid_connect/userinfo' => 'openid_connect/user_info#show'
+  get '/openid_connect/certs' => 'openid_connect/certs#index', as: :old_openid_connect_certs
+  post '/openid_connect/token' => 'openid_connect/token#create', as: :old_openid_connect_token
+  get '/openid_connect/userinfo' => 'openid_connect/user_info#show',
+      as: :old_openid_connect_userinfo
```

Comment:
The purpose of keeping these backwards compatible would be easier communication with the teams currently developing against the API, and a little bit of leeway in terms of deploy schedules for us and them to get them to switch?